react/components/SideBar.tsx

```diff
@@ -42,6 +43,11 @@ const SideBar: FC = () => {
             <VTEXPink />
           </Link>
         </div>
+        <div>
```

Comment:
Is this `<div>` really necessary?
I suggest trying to pass `db` token to `Link` below

Reply:
Nope, removed it! Also added `db`